app/products/product.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, OneToMany, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { CartItem } from '../cart/cartItem.entity';
+
+@Entity({ name: 'products' })
+export class Product {
+  @PrimaryGeneratedColumn('uuid')
+  id: UUID;
+
+  @Column('text')
+  title: string;
+
+  @Column('text')
+  description: string;
+
+  @Column('decimal')
+  price: number;
+
+  @OneToMany(() => CartItem, (cartItems) => cartItems.product)
+  cartItems: CartItem[];
```

Comment:
Why do you need this field in product model?

Reply:
To bind product and cart items. TypeORM requires specifying this decorator on both entities